rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -1111,66 +1112,49 @@ def test_taxable_acquisition(accountant: Accountant) -> None:
 @pytest.mark.parametrize(('db_settings', 'expected_pnls'), [
     (
         {'cost_basis_method': CostBasisMethod.FIFO, 'include_fees_in_cost_basis': False},
-        [ZERO, ZERO, FVal(-10), FVal(3500), ZERO, FVal(-10), ZERO, ZERO, FVal(-10), FVal(1600), ZERO, FVal(-10)],  # noqa: E501
+        [ZERO, FVal(-10), ZERO, FVal(3500), ZERO, FVal(-10), ZERO, FVal(-10), ZERO, FVal(1600), ZERO, FVal(-10)],  # noqa: E501
```

Comment:
why did this change?

Reply:
`_process_swap()` takes the spend & fee before taking the receive so that's the reason for that change.

Reply:
Shouldn't you then change process swap? PReserve the order? We need spend, receive then fee.